src/MarkdownTextInput.web.tsx

```diff
@@ -236,6 +236,35 @@ const MarkdownTextInput = React.forwardRef<MarkdownTextInput, MarkdownTextInputP
       [parser, parseText, processedMarkdownStyle],
     );
 
+    const format = useCallback(
+      (target: MarkdownTextInputElement, parsedText: string, cursorPosition: number, formatType: string): ParseTextResult => {
+        if (!contentSelection.current) {
+          return {
+            text: '',
+            cursorPosition: 0,
+          };
+        }
+        let markdown;
+        switch (formatType) {
+          case 'formatBold':
+            markdown = '*';
+            break;
+          case 'formatItalic':
+            markdown = '_';
+            break;
+          default:
+            markdown = '';
```

Comment:
I think this state is unreachable so we could just throw an error here.

Reply:
Is it better if I add underline as the default value?
```
switch (formatCommand) {
  case 'formatBold':
    formatType = 'bold';
    break;
  case 'formatItalic':
    formatType = 'italic';
    break;
  default:
    formatType = 'underline';
    break;
}
```